src/runtime/manifest-parser.peg

```diff
@@ -339,6 +339,16 @@ ParticleClaimStatement
   }
 
 ParticleClaimExpression
+  = first:ParticleClaim rest:(whiteSpace 'and' whiteSpace ParticleClaim)*
+  {
+    if (rest.length === 0) {
+      return [first] as AstNode.ParticleClaimExpression;
+    }
+    const claims: AstNode.ParticleClaim[] = [first, ...rest.map(item => item[3])];
+    return claims as AstNode.ParticleClaimExpression;
```

Comment:
This whole function can be replaced with:

```
return [first, ...rest.map(item => item[3])] as AstNode.ParticleClaimExpression;
```

(I don't think you need the if-statement, it'll just be an empty list, which `...rest.map()` will handle correctly)

Reply:
Done. Thanks.